### PR TITLE
Update services

### PR DESCRIPTION
**Pull request by confleux**
- Added services ⟶ list ⟶ description (software, github, docs)
- Added bitcoind
- Move eth indexer to services

### CODE REVIEW

**Comment by adamant-al** [comment]
Updated description.

File: assets/general/ethereum/info.json [hunk 1]
```diff
@@ -96,6 +118,31 @@
         "onScreenUpdateInterval": 10000,
         "threshold": 5
       }
+    },
+    "services": {
+      "description": {
+        "software": "ETH-transactions-storage",
+        "github": "https://github.com/Adamant-im/ETH-transactions-storage",
+        "docs": "https://github.com/Adamant-im/ETH-transactions-storage?tab=readme-ov-file#api-request-examples"
```
Comment (bludnic) [inline assets/general/ethereum/info.json:126]: Update the OpenAPI schema:

1. Create `ServiceDescription` in the [components](https://github.com/Adamant-im/adamant-wallets/blob/chore/services/specification/openapi.json#L8) section
2. Add the `ServiceDescription` component to [services](https://github.com/Adamant-im/adamant-wallets/blob/chore/services/specification/openapi.json#L243)

**Review by bludnic — CHANGES_REQUESTED** [review]

File: assets/general/ethereum/info.json (hunk 1 above)
Comment (bludnic) [reply assets/general/ethereum/info.json:126]: It's important to keep the OpenAPI schema synced every time a new property is added. The PWA generates TS types from it.

Comment (bludnic) [reply assets/general/ethereum/info.json:126]: You can validate and visualise the schema on https://editor.swagger.io

**Comment by RealGoodProgrammer** [comment]
This branch will be included in the next release

**Comment by bludnic** [comment]
@confleux Merge `dev` and fix the conflicts. Resolve opened issues.

**Comment by adamant-al** [comment]
Include new nodes and remove outdated.

**Comment by adamant-al** [comment]
@confleux 

Merge dev and fix the conflicts. Resolve opened issues.
Update ethnodes, btcnodes.

**Comment by adamant-al** [comment]
@bludnic @just-software-dev 
Review.

**Review by bludnic — APPROVED** [review]

**Pull request merged**